packages/@aws-cdk/aws-stepfunctions-tasks/test/codebuild/integ.start-build.ts

```diff
@@ -16,8 +16,17 @@ class StartBuildStack extends cdk.Stack {
   constructor(scope: cdk.App, id: string, props: cdk.StackProps = {}) {
     super(scope, id, props);
 
+    const owner = 'aws';
+    const repo = 'aws-cdk';
+    const source = codebuild.Source.gitHub({
+      owner,
+      repo,
+      branchOrRef: 'master',
+    });
+
```

Comment:
Can you actually create a new integ test, that uses overrides, instead of changing the existing one?